ImportExtension/AddImportOperation.swift

```diff
@@ -41,6 +47,30 @@ class AddImportOperation {
         }
 
         guard self.buffer.canIncludeImportString(importString, atLine: line) else {
+
+            if let potentialRemovalInstructionsString = self.buffer.lines[selectionLine + 1] as? String {
+                
+                if (potentialRemovalInstructionsString.trimmingCharacters(in: CharacterSet.newlines) == AddImportOperationConstants.doubleImportRemoveInstructionsString)
+                {
+                    self.buffer.lines.removeObjects(in: NSMakeRange(selectionLine, 2))
+                    
+                    return
+                }
+                
+            }
+            
+            NSBeep()
```

Comment:
Hey good call!

![untitled 2](https://cloud.githubusercontent.com/assets/676023/19435362/8875cc62-946a-11e6-9232-9e951e11a85a.png)

I’d have to rewrite that to make use of the alert.